netdata.spec.in

```diff
@@ -1,90 +1,100 @@
+#
+# Conditional build:
+%bcond_without	systemd		# systemd
+%bcond_without	nfacct		# build with nfacct plugin
```

Comment:
[it is so](https://github.com/firehol/netdata/pull/186/files/7154c1bad0b0aa18eddaa5b917104f108561c169#r58607212)

[undefine the with_foo](https://github.com/glensc/netdata/blob/rpmspec/netdata.spec.in#L6-L9)